tests/python/test_ndarray.py

```diff
@@ -603,7 +607,7 @@ def test_arg_not_match():
     _test_arg_not_match()
 
 
-@ti.test(arch=ti.opengl, ndarray_use_torch=True)
+@ti.test(arch=ti.vulkan, ndarray_use_torch=True)
```

Comment:
just to confirm, this is vulkan instead of opengl?

Reply:
ah yes, since we supported opengl I changed it to vulkan which is still not supported yet. :P Thanks will change the test name!